java/com/google/flatbuffers/FlatBufferBuilder.java

```diff
@@ -368,6 +368,39 @@ public int endVector() {
     /// @endcond
 
    /**
+     * Create a vector of tables.
+     *
+     * @param offsets Offsets of the tables.
+     * @return Returns offset of the vector.
+     */
+    public int createVectorOfTables(int[] offsets) {
+        notNested();
+        startVector(4, offsets.length, 4);
+        for(int i = offsets.length - 1; i >= 0; i--) addOffset(offsets[i]);
+        return endVector();
+    }
+
+    /**
+     * Create a vector of sorted by the key tables.
+     *
+     * @param type Type of the tables.
+     * @param offsets Offsets of the tables.
+     * @return Returns offset of the sorted vector.
+     */
+    public <T> int createSortedTableVector(Class<T> type, int[] offsets) {
+        try{
+            return (int)type.getMethod("createMySortedTableVector", FlatBufferBuilder.class, ByteBuffer.class, int[].class).invoke(null, this, bb, offsets);
```

Comment:
Since it is only needed to create sorted vectors (not normal vectors), and since you likely already have such an object from creating the elements of the vector, I think this is ok.